packages/commonwealth/client/scripts/views/components/component_kit/cw_avatar_group.tsx

```diff
@@ -18,21 +18,30 @@ export type ProfileWithAddress = MinimumProfile & {
 type AvatarGroupProps = {
   profiles: ProfileWithAddress[];
   communityId: string;
+  totalProfiles?: number;
 };
 
-export const CWAvatarGroup = ({ profiles, communityId }: AvatarGroupProps) => {
+export const CWAvatarGroup = ({
+  profiles,
+  communityId,
+  totalProfiles,
+}: AvatarGroupProps) => {
   if (!profiles || profiles?.filter((p) => !!p && p.Addresses).length === 0)
     return;
 
+  const maxProfileAvatars = 4;
+
   const truncatedProfiles = profiles
     .filter((p) => !!p && p.Addresses)
-    .slice(0, 4)
+    .slice(0, maxProfileAvatars)
     .reverse();
 
   const count = profiles.length - 4;
   let countText;
 
-  if (count > 5) {
+  if (totalProfiles && totalProfiles > maxProfileAvatars) {
```

Comment:
maybe a nit here, but we don't need to check if totalProfiles exists. If for example totalprofiles = null or undefined we have:
`undefined > maxProfileAvatars === false`